### PR TITLE
batch_system: fix build when building the crate alone

### LINKED ISSUE #5209: bench raftstore::store::fsm::router::tests::bench_send failed
## Bug Report

**What version of TiKV are you using?**

Commit: 995257ddb1c926cfce638663b66c92809064eedd

**What operating system and CPU are you using?**

MacOSX

**What did you do?**

```
RUST_BACKTRACE=1 cargo bench --no-default-features --features " jemalloc sse failpoints"
```

**What did you expect to see?**

Bench passed

**What did you see instead?**

```
failures:

---- raftstore::store::fsm::router::tests::bench_send stdout ----
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: "Full(..)"', src/libcore/result.rs:999:5
stack backtrace:
   0: std::panicking::default_hook::{{closure}}
   1: std::panicking::default_hook
   2: std::panicking::rust_panic_with_hook
   3: std::panicking::continue_panic_fmt
   4: rust_begin_unwind
   5: core::panicking::panic_fmt
   6: core::result::unwrap_failed
   7: core::ops::function::FnOnce::call_once
   8: std::panicking::try::do_call
   9: __rust_maybe_catch_panic
  10: test::run_test
  11: test::run_tests_console
  12: test::test_main
  13: test::test_main_static
  14: std::rt::lang_start::{{closure}}
  15: std::panicking::try::do_call
  16: __rust_maybe_catch_panic
  17: std::rt::lang_start_internal
  18: main


failures:
    raftstore::store::fsm::router::tests::bench_send
```

### PR DESCRIPTION
**Pull request by yiwu-arbug**
Signed-off-by: Yi Wu <yiwu@pingcap.com>

### What problem does this PR solve?

Issue Number: close #5209

Problem Summary:

Fixing build error when building `batch_system` crate alone, i.e. want to run the bench under the crate by:
```
cargo bench -p batch-system --feature protobuf-codec
```

### What is changed and how it works?

What's Changed: updating features flag in Cargo.toml.

### Related changes

N/A

### Check List 

Tests

- Manually run `cargo bench -p batch-system --feature protobuf-codec`

### Release note 

- No release notes.

### CODE REVIEW

File: components/tikv_util/Cargo.toml [hunk 1]
```diff
@@ -45,7 +45,7 @@ sysinfo = "0.15"
 tikv_alloc = { path = "../tikv_alloc" }
 collections = { path = "../collections" }
 time = "0.1"
-tokio = { version = "0.2", features = ["rt-util"] }
+tokio = { version = "0.2", features = ["rt-util", "rt-core"] }
```
Comment (BusyJay) [inline components/tikv_util/Cargo.toml:48]: Why rt-core is necessary?

**Review by hunterlxt — APPROVED** [review]
LGTM

File: components/tikv_util/Cargo.toml (hunk 1 above)
Comment (hunterlxt) [reply components/tikv_util/Cargo.toml:48]: I introduced `tokio::task::LocalSet` several months ago

**Review by BusyJay — APPROVED** [review]

**Review by NingLin-P — APPROVED** [review]

**Pull request merged**